From 247edebe8f922f097b310389c759c8d46ddd44cb Mon Sep 17 00:00:00 2001
From: Zane Shelley <zshelle@us.ibm.com>
Date: Mon, 16 Apr 2018 14:49:44 -0500
Subject: [PATCH 3/7] PRD: set INTCQFIR[27:29] to threshold_and_mask_self

Change-Id: I20ad86756b80964ed635c947d94273ea6899b576
CQ: SW423945
Backport: release-op910
Backport: release-fips910
Reviewed-on: http://ralgit01.raleigh.ibm.com/gerrit1/57286
Reviewed-by: Benjamin J. Weisenbeck <bweisenb@us.ibm.com>
Reviewed-by: Caleb N. Palmer <cnpalmer@us.ibm.com>
Reviewed-by: Brian J. Stegmiller <bjs@us.ibm.com>
Reviewed-by: Matt Derksen <mderkse1@us.ibm.com>
Tested-by: Jenkins Server <pfd-jenkins+hostboot@us.ibm.com>
Reviewed-by: Zane C. Shelley <zshelle@us.ibm.com>
Reviewed-on: http://ralgit01.raleigh.ibm.com/gerrit1/57340
CI-Ready: Zane C. Shelley <zshelle@us.ibm.com>
Tested-by: Jenkins OP Build CI <op-jenkins+hostboot@us.ibm.com>
Tested-by: Jenkins OP HW <op-hw-jenkins+hostboot@us.ibm.com>
---
 src/usr/diag/prdf/common/plat/p9/p9_cumulus.rule | 6 +++---
 src/usr/diag/prdf/common/plat/p9/p9_nimbus.rule  | 6 +++---
 2 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/src/usr/diag/prdf/common/plat/p9/p9_cumulus.rule b/src/usr/diag/prdf/common/plat/p9/p9_cumulus.rule
index 69b638198..f8c36f52e 100644
--- a/src/usr/diag/prdf/common/plat/p9/p9_cumulus.rule
+++ b/src/usr/diag/prdf/common/plat/p9/p9_cumulus.rule
@@ -5927,17 +5927,17 @@ group gINTCQFIR filter singlebit, cs_root_cause
     /** INTCQFIR[27]
      *  INT_CQ_FIR_WRQ_OP_HANG:
      */
-    (rINTCQFIR, bit(27)) ? self_th_1;
+    (rINTCQFIR, bit(27)) ? threshold_and_mask_self;
 
     /** INTCQFIR[28]
      *  INT_CQ_FIR_RDQ_OP_HANG:
      */
-    (rINTCQFIR, bit(28)) ? self_th_1;
+    (rINTCQFIR, bit(28)) ? threshold_and_mask_self;
 
     /** INTCQFIR[29]
      *  INT_CQ_FIR_INTQ_OP_HANG:
      */
-    (rINTCQFIR, bit(29)) ? self_th_1;
+    (rINTCQFIR, bit(29)) ? threshold_and_mask_self;
 
     /** INTCQFIR[30]
      *  INT_CQ_FIR_RDQ_DATA_HANG:
diff --git a/src/usr/diag/prdf/common/plat/p9/p9_nimbus.rule b/src/usr/diag/prdf/common/plat/p9/p9_nimbus.rule
index 6dfdb81af..b06d9ff40 100644
--- a/src/usr/diag/prdf/common/plat/p9/p9_nimbus.rule
+++ b/src/usr/diag/prdf/common/plat/p9/p9_nimbus.rule
@@ -5907,17 +5907,17 @@ group gINTCQFIR filter singlebit, cs_root_cause
     /** INTCQFIR[27]
      *  INT_CQ_FIR_WRQ_OP_HANG:
      */
-    (rINTCQFIR, bit(27)) ? self_th_1;
+    (rINTCQFIR, bit(27)) ? threshold_and_mask_self;
 
     /** INTCQFIR[28]
      *  INT_CQ_FIR_RDQ_OP_HANG:
      */
-    (rINTCQFIR, bit(28)) ? self_th_1;
+    (rINTCQFIR, bit(28)) ? threshold_and_mask_self;
 
     /** INTCQFIR[29]
      *  INT_CQ_FIR_INTQ_OP_HANG:
      */
-    (rINTCQFIR, bit(29)) ? self_th_1;
+    (rINTCQFIR, bit(29)) ? threshold_and_mask_self;
 
     /** INTCQFIR[30]
      *  INT_CQ_FIR_RDQ_DATA_HANG:
-- 
2.16.2.windows.1

